pypit/data/settings/settings.kast_blue

```diff
@@ -28,7 +28,7 @@ check 01.NAXIS 2                       # THIS IS A MUST! It performs a standard
 check 01.DSENSOR Fairchild CCD 3041 2Kx2K          # Check the CCD name (replace any spaces with underscores)
 
 ### Keyword Identifiers
-keyword time 01.TSEC                   # The time stamp of the observation (i.e. decimal MJD)
+keyword time 01.TSEC                   # A time stamp of the observation; not sure what this is really..
```

Comment:
Looking back through the code, I remember that I used this only to find the time difference between science frames and calibration frames. Ideally, this should be an MJD of the midpoint of observation, or in the ISOT format.